henchman/helpers.go

```diff
@@ -21,6 +24,16 @@ func MergeMap(src map[interface{}]interface{}, dst map[interface{}]interface{},
 	}
 }
 
+func MergeLogrusFields(src log.Fields, dst log.Fields, override bool) {
```

Comment:
This will require making a customMap interface.  Similar to this guys issue.
https://groups.google.com/forum/#!topic/golang-nuts/Ia6Q6ztiJXI